addons/im_livechat/static/src/new/chat_window/chat_window_patch.xml

```diff
@@ -10,5 +10,19 @@
             />
            <t t-else="">$0</t>
         </xpath>
+        <xpath expr="//Composer" position="replace">
+            <t t-if="chatbotService.inputEnabled">$0</t>
+            <t t-else="">
+                <span class="bg-200 py-1 text-center fst-italic" t-esc="chatbotService.inputDisabledText"/>
+            </t>
+        </xpath>
+        <xpath expr="//i[hasclass('fa-close')]/ancestor::div[hasclass('o-mail-ChatWindow-command')]" position="before">
```

Comment:
Better defining a `name` in template, so the patch is simpler and safer.
Don't over-engineer it: there's only one patch to add a new command in chat window header.
If we see future other uses, we'll make a registry specifically to add actions there.